finality-provider/cmd/fpd/daemon/commit_pr.go

```diff
@@ -0,0 +1,79 @@
+package daemon
+
+import (
+	"fmt"
+	"path/filepath"
+	"strconv"
+
+	bbntypes "github.com/babylonlabs-io/babylon/types"
+	fpcmd "github.com/babylonlabs-io/finality-provider/finality-provider/cmd"
+	fpcfg "github.com/babylonlabs-io/finality-provider/finality-provider/config"
+	"github.com/babylonlabs-io/finality-provider/finality-provider/service"
+	"github.com/babylonlabs-io/finality-provider/log"
+	"github.com/babylonlabs-io/finality-provider/util"
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/spf13/cobra"
+)
+
+// CommandCommitPubRand returns the commit-pubrand command by connecting to the fpd daemon.
+func CommandCommitPubRand() *cobra.Command {
+	var cmd = &cobra.Command{
+		// TODO: add --start-height as an optional flag
+		Use:     "unsafe-commit-pubrand [fp-eots-pk-hex] [block-height]",
+		Aliases: []string{"unsafe-cpr"},
+		Short:   "[UNSAFE] Manually trigger public randomness commitment for a finality provider",
+		Long: `[UNSAFE] Manually trigger public randomness commitment for a finality provider. ` +
+			`WARNING: this can drain the finality provider's balance if the block-height is too high.` +
+			`Note: if there is no pubrand committed before, it will only commit the pubrand for the target block-height.`,
```

Comment:
In this case, we probably need the user to specify `--start-height` as this is probably a mistake. Making a commit for a single height does not make much sense unless the user explicity specifies it

Reply:
good catch. this docstring needs to be fixed. it's no longer the case with the new updates in this PR